core/src/main/java/org/infinispan/util/CoreBlockHoundIntegration.java

```diff
@@ -122,5 +123,9 @@ private static void questionableMethodsAllowedToBlock(BlockHound.Builder builder
 
       // This can block if there is a store otherwise it won't block
       builder.allowBlockingCallsInside(CacheMgmtInterceptor.class.getName(), "getNumberOfEntries");
+
+      // Unfortunately retrieving the protobuf schema reads from a separately generated file - We hope this can be changed
```

Comment:
We also start/stop caches and register schemas at runtime. Having this as a String in the file also saves a lot of heartache for Infinispan Quarkus as well. It would be a lot simpler and nicer to have it in there eventually, but that requires changes in protostream.